internal/provider/resources/webhooks_test.go

```diff
@@ -131,6 +134,23 @@ func testAccCheckWebhookExists(webhookResourceName string, webhook *api.Webhook)
 	}
 }
 
+func testAccCheckWebhookEndpoint(webhookResourceName string, webhook *api.Webhook) resource.TestCheckFunc {
```

Comment:
it's worth calling out here, since i couldn't verify this behavior from their docs + only found it through testing the actual control flow

basically, there's a pattern in the testing docs where you can instantiate a pointer outside of the test steps, hydrate it with a custom test function, and then reference it again in the subsequent function

https://developer.hashicorp.com/terraform/plugin/testing/acceptance-tests/teststep#custom-check-functions

however, the nuance is that these custom functions seem to run well after any pre-built check function on the `resource.` object, eg. `resource.TestCheckResourceAttr` - so you can't run these resource checks against the hydrated value, because the pointer will still be a zero value

this answers the issue raised here, so closing this PR will resolve #120